base/acme/src/main/java/org/dogtagpki/acme/database/PostgreSQLDatabase.java

```diff
@@ -110,6 +113,39 @@ public void connect() throws Exception {
         }
     }
 
+    /**
+     * This method will create the tables if they do not exist.
+     */
+    public void setup() throws Exception {
+
+        logger.info("Setting up database");
+
+        String filename = "/usr/share/pki/acme/database/postgresql/create.sql";
+        String content = new String(Files.readAllBytes(Paths.get(filename)));
+
+        String[] statements = content.split(";");
```

Comment:
Yuck. I guess there's no other way of doing this with JDBC as far as I can tell. I think having it in the initialization script is as good as anything, so I guess we're stuck with the initialization and parsing logic here. I'd maybe take the alternative route of a separate CLI to setup databases in general (so this occurs once every time the admin executes it instead of once every start up), but I understand why this is attractive for containers.

Reply:
I was hoping to find a SQL parser in PostgreSQL JDBC driver, but there's none, so the best solution I have now is splitting the SQL file like above.

Right now the `setup()` will create the initial tables, but in the future we might use it to upgrade the tables as well. So when the container image is updated, it will automatically upgrade the database too on restart.

Reply:
Yeah that's fair. My one concern that I'd need to test is what if we include comments in `create.sql` later? Will this try and execute the comments? And there can't be a `;` in the comment... Hmmm...

Reply:
I tried adding these two comment styles and they actually work just fine:
```
-- comment
```
and
```
/* comment */
```
But yeah, they can't contain a `;` for now.